bin/setup.sh

```diff
@@ -46,7 +47,7 @@ wait_for_plugin_registration () {
 }
 
 load_bundled_data ( ) {
-  __skriptz=$(ls -1 ${COOPR_HOME}/provisioner/lib/provisioner/worker/plugins/*/*/load-bundled-data.sh 2>&1)
+  __skriptz=$(ls -1 ${COOPR_PROVISIONER_PLUGIN_DIR}/*/*/load-bundled-data.sh 2>&1)
```

Comment:
So, these are not the same, right?

Goes from:
 ``${COOPR_HOME}/provisioner/lib/provisioner/worker/plugins``

to:
``${COOPR_HOME}/provisioner/lib/worker/plugins``

drops ``provisioner`` in the middle?